modules/htmlutil/html.go

```diff
@@ -31,6 +32,9 @@ func ParseSizeAndClass(defaultSize int, defaultClass string, others ...any) (int
 }
 
 func HTMLFormat(s template.HTML, rawArgs ...any) template.HTML {
+	if !strings.Contains(string(s), "%") || len(rawArgs) == 0 {
+		panic("HTMLFormat requires at least one argument, don't abuse it")
```

Comment:
```suggestion
		panic("HTMLFormat requires one or more argument")
```

More neutral message